static/docs/user-guide/index.md

```diff
@@ -0,0 +1,33 @@
+# User Guide
+
+This section describes the main DVC concepts and features comprehensively,
+explaining when and how to use them, as well as connections between them. These
+guides don't focus on specific scenarios, but have a general scope – like a user
+manual. Their topics range from more technical basics, which impact more parts
+of DVC, to more advanced things you can do. We also include a few guides related
+to contributing to this [open-source project](https://github.com/iterative/dvc).
+
+- [DVC Files and Directories](/doc/user-guide/dvc-files-and-directories)
```

Comment:
I hope that we'll have a proper third level soon to cluster some of these by topics. Not for this PR for sure.

Reply:
Good idea! Moved your remarks to https://github.com/iterative/dvc.org/issues/654#issuecomment-536330090

Reply:
👍